docs/cases/cases-manage.asciidoc

```diff
@@ -0,0 +1,173 @@
+
+
+[role="screenshot"]
+image::images/cases-ui-open.png[Shows an open case]
+
+[float]
+[[cases-ui-manage]]
+== Manage existing cases
+
+From the Cases page, you can search existing cases and filter them by severity, tags, reporter, and status: *open*, *in-progress*, or *closed*. General case metrics, including how long it takes to close cases, are provided above the table.
+
+[role="screenshot"]
+image::images/cases-home-page.png[Case UI Home]
+
+To explore a case, click on its name. You can then:
+
+* <<cases-summary>>
+* Add and <<cases-manage-comments, manage comments>>
++
+TIP: Comments can contain Markdown sytax. For syntax assistance, click the Markdown button (image:images/markdown-icon.png[Click markdown icon,17,17]) in the bottom right corner of the comment area.
+
+* <<cases-examine-alerts, Examine alerts attached to the case>>
+* <<cases-lens-visualization>>
+* Modify the case's description
+* Add a <<cases-ui-integrations, connector>> (if you did not select one while creating the case)
+* Send updates to external systems (if external connections are configured)
+* Close the case or reopen it
+* Edit tags
+* Update the case's severity
+* Refresh the case to retrieve the latest updates
+
+[float]
+[[cases-summary]]
+=== Review the case summary
+
+Click on an existing case to access the case summary. The case summary, located under the case title, contains metrics that summarize alert information and response times. These metrics update when you attach additional unique alerts to the case, add more connectors, or modify the case's status:
+
+* **Total alerts**: Total number of unique alerts attached to the case.
+* **Associated users**: Total number of unique users that are represented in the attached alerts.
+* **Associated hosts**: Total number of unique hosts that are represented in the attached alerts.
+* **Total connectors**: Total number of connectors that have been added to the case.
+* **Case created**: Date and time that the case was created.
+* **Open duration**: Time elapsed since the case was created.
+* **In progress duration**: How long the case has been in the `In progress` state.
+* **Duration from creation to close**: Time elapsed from when the case was created to when it was closed.
+
+[role="screenshot"]
+image::images/cases-summary.png[Shows you a summary of the case]
+
+[float]
+[[cases-manage-comments]]
+=== Manage case comments
+To edit, delete, or quote a comment, click the *More actions* menu (*…​*) and select the appropriate option.
+
+[role="screenshot"]
+image::images/cases-manage-comments.png[Shows you a summary of the case]
+
+[float]
+[[cases-examine-alerts]]
+=== Examine alerts attached to a case
+
+preview::[]
+
+To explore the alerts attached to a case, click the *Alerts* tab. In the table, alerts are organized from oldest to newest. To <<view-alert-details, view alert details>>, click the *View details* button.
+
+[role="screenshot"]
+image::images/cases-alert-tab.gif[Shows you the Alerts tab]
```

Comment:
Yeah, I see your point. I was going back and forth on the gif as well because the alert details flyout will likely change in a future release. Omitting it here and keeping the focus on the table makes sense.

Reply:
Replaced with a static image of the **Alerts** tab.